arkane/log.py

```diff
@@ -77,3 +83,36 @@ def loadNegativeFrequency(self):
         """
         raise NotImplementedError("loadGeometry is not implemented for the Log class")
 
+    def get_optical_isomers_and_symmetry_number(self):
+        """
+        This method uses the symmetry package from RMG's QM module
+        and returns a tuple where the first element is the number
+        of optical isomers and the second element is the symmetry number.
+        """
+        coordinates, atom_numbers, _ = self.loadGeometry()
+        unique_id = '0'  # Just some name that the SYMMETRY code gives to one of its jobs
+        scr_dir = os.path.join(os.path.abspath('.'), str('scratch'))  # Scratch directory that the SYMMETRY code writes its files in
+        if not os.path.exists(scr_dir):
+            os.makedirs(scr_dir)
+        try:
+            qmdata = QMData(
+                groundStateDegeneracy=1,  # Only needed to check if valid QMData
+                numberOfAtoms=len(atom_numbers),
+                atomicNumbers=atom_numbers,
+                atomCoords=(coordinates, str('angstrom')),
+                energy=(0.0, str('kcal/mol'))  # Only needed to avoid error
+            )
+            settings = type(str(''), (), dict(symmetryPath=str('symmetry'), scratchDirectory=scr_dir))()  # Creates anonymous class
+            pgc = PointGroupCalculator(settings, unique_id, qmdata)
+            pg = pgc.calculate()
+            if pg is not None:
+                optical_isomers = 2 if pg.chiral else 1
+                symmetry = pg.symmetryNumber
+                logging.debug("Symmetry algorithm found {0} optical isomers and a symmetry number of {1}".format(optical_isomers,symmetry))
+            else:
+                logging.warning("Symmetry algorithm errored when computing optical isomers. Setting to 1")
```

Comment:
I think this should be a `logging.error` level, and the text should tell the user to try re-running and supply the missing values (unless we want to raise an error?)
Also, the current message only refers to opt isomers, symmetry could be problematic as well

Reply:
I wonder how making this an error might cause issues when integrating with ARC (or other automated software generation). I guess we just have to integrate the code in a way such that a factor of two error does not destroy the whole RMG run.

Reply:
Changed to logging.error